### DISCUSSION TITLE
[Question]: How to automatically update and upgrade this item on the server

### DISCUSSION THREAD

**Discussion by MackDing** [danny-avila/LibreChat#1927] [Q&A]
### What is your question?

How to automatically update and upgrade this item on the server

### More Details

I want to use the latest version of the app every time

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
I assume you've deployed LibreChat with docker? You could use an external tool like Watchtower to monitor and automatically update LibreChat.
https://github.com/containrrr/watchtower
